core/types/tx_blob.go

```diff
@@ -65,9 +65,9 @@ func (s BlobTxSidecars) EncodeIndex(i int, w *bytes.Buffer) {
 
 // BlobTxSidecar contains the blobs of a blob transaction.
 type BlobTxSidecar struct {
-	Blobs       []kzg4844.Blob       // Blobs needed by the blob pool
-	Commitments []kzg4844.Commitment // Commitments needed by the blob pool
-	Proofs      []kzg4844.Proof      // Proofs needed by the blob pool
+	Blobs       []kzg4844.Blob       `json:"blobs"`       // Blobs needed by the blob pool
```

Comment:
when marshal
need to transfer to hex? otherwise may not suitable for human to see

not quite sure, just check

Reply:
yes, need to transfer to hex. Refer to this: https://ethereum.github.io/beacon-APIs/#/Beacon/getBlobSidecars